Assets/HoloToolkit/BuildAndDeploy/Editor/BuildDeployWindow.cs

```diff
@@ -213,17 +220,28 @@ private void OnGUI()
             EditorGUILayout.BeginHorizontal();
             GUILayout.FlexibleSpace();
             EditorGUILayout.EndHorizontal();
+            EditorGUILayout.BeginHorizontal();
 
             // Build directory (and save setting, if it's changed)
             string curBuildDirectory = BuildDeployPrefs.BuildDirectory;
-            string newBuildDirectory = EditorGUILayout.TextField(GUIHorizSpacer + "Build directory", curBuildDirectory);
+            string newBuildDirectory = EditorGUILayout.TextField(new GUIContent(GUIHorizontalSpacer + "Build Directory", "It's recommended to use UWP"), curBuildDirectory);
 
             if (newBuildDirectory != curBuildDirectory)
             {
                 BuildDeployPrefs.BuildDirectory = newBuildDirectory;
                 curBuildDirectory = newBuildDirectory;
             }
 
+            GUI.enabled = Directory.Exists(BuildDeployPrefs.AbsoluteBuildDirectory);
+
+            if (GUILayout.Button("Open Build Directory"))
+            {
+                Process.Start("explorer.exe", "/f /open," + BuildDeployPrefs.AbsoluteBuildDirectory);
```

Comment:
This will open a new explorer window every time. If we used `Process.Start(BuildDeployPrefs.AbsoluteBuildDirectory)` instead, I think it would re-use the same explorer window and just bring it to the front.

Reply:
That was the intention to open a new window each time.

I usually have specific explorer windows open, and would be frustrated if one of them were to change on me unexpectedly.

Reply:
Sorry, to clarify the behavior is that it will only reuse the window if one is already open to that exact directory. It won't reuse any windows that are opened to different directories. You can test out the behavior by typing directory names in the Win+R run dialog.

Reply:
I also wanted to use `/f` to give the window focus because without it the editor stays on top.

Reply:
nvm, seems to work fine.

Reply:
Ah I see, well if that is necessary than that sounds good. I couldn't find any actual documentation of the `/f` flag online, do you have a reference for that?